spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestReadProjection.java

```diff
@@ -36,23 +38,19 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
 import org.apache.iceberg.types.Comparators;
 import org.apache.iceberg.types.Types;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
 
+@ExtendWith(ParameterizedTestExtension.class)
 public abstract class TestReadProjection {
-  final String format;
```

Comment:
shouldn't this be marked with `@Parameter`?

Reply:
`format` isn't actively being used anywhere in the file. It's only in the commented code of a few tests.
So I'm not sure if its needed